ROS/osr/src/roboclaw_wrapper.py

```diff
@@ -63,7 +63,7 @@ def run(self):
         while not rospy.is_shutdown():
 
             while self.mutex and not rospy.is_shutdown():
-                mutex_rate.sleep()
+                time.sleep(0.001)
```

Comment:
Don't use `time` in ROS loops as they are blocking and do not leave space for ROS to handle callbacks!